app/services/most_popular_content.rb

```diff
@@ -0,0 +1,35 @@
+class MostPopularContent
+  attr_reader :content_id
+
+  def initialize(content_id:)
+    @content_id = content_id
+  end
+
+  def self.fetch(content_id:)
+    new(content_id: content_id).fetch
+  end
+
+  def fetch
+    search_response
+      .documents
+      .sort_by { |document| document.title }
+  end
+
+private
+
+  def hide_from_most_popular
```

Comment:
I'm passing this as a `reject_` filter, which means it will always return 7 results if there are 7 that match the query.